src/server/auth/cmds/cmds.go

```diff
@@ -79,8 +94,141 @@ func LoginCmd() *cobra.Command {
 	return login
 }
 
+// CheckCmd returns a cobra command that sends an "Authorize" RPC to Pachd, to
+// determine whether the specified user has access to the specified repo.
+func CheckCmd() *cobra.Command {
+	check := &cobra.Command{
+		Use:   "check (none|reader|writer|owner) repo",
+		Short: "Check whether you have reader/writer/etc-level access to 'repo'",
+		Long: "Check whether you or another user has a reader/writer/etc-level " +
+			"access to 'repo'. For " +
+			"example, 'pachctl auth check reader private-data' prints " +
+			"\"true\" if the you have at least \"reader\" access " +
+			"to the repo \"private-data\" (you could be a reader, writer, " +
+			"or owner). Unlike. `pachctl get-acl`, you do not need to have access " +
+			"to 'repo' to discover your own acess level.",
+		Run: cmdutil.RunFixedArgs(3, func(args []string) error {
+			scope, err := auth.ParseScope(args[0])
+			if err != nil {
+				return err
+			}
+			repo := args[1]
+			c, err := client.NewOnUserMachine(true, "user")
+			if err != nil {
+				return fmt.Errorf("could not connect: %s", err.Error())
+			}
+			resp, err := c.AuthAPIClient.Authorize(
+				c.RequestCtx(),
+				&auth.AuthorizeRequest{
+					Repo:  &pfs.Repo{Name: repo},
+					Scope: scope,
+				})
+			if err != nil {
+				return err
+			}
+			fmt.Printf("%t\n", resp.Authorized)
+			return nil
+		}),
+	}
+	return check
+}
+
+// GetCmd returns a cobra command that gets either the ACL for a Pachyderm
+// repo or another user's scope of access to that repo
+func GetCmd() *cobra.Command {
```

Comment:
Is this command going to look like `pachctl get`?  That sounds a little misleading.  Maybe `get-acl`?